<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stickyNote</class>
 <widget class="QWidget" name="stickyNote">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sticky Note</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/sticky-notes.png</normaloff>../icons/sticky-notes.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#noteTitleFrame{
	background-color: rgb(239, 213, 165);
	border-top-right-radius: 8px;
	border-top-left-radius: 8px;
}

#noteMainFrame{
	background-color: #fae0ad;
	border-bottom-right-radius: 8px;
	border-bottom-left-radius: 8px;
}
</string>
  </property>
  <widget class="QFrame" name="noteTitleFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="closeNote">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>0</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: none;
	border-top-right-radius: 8px;
}
QPushButton:hover{
	background-color: rgb(213, 190, 147);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/closebutton.png</normaloff>../icons/closebutton.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="minimizeNote">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: none;
}
QPushButton:hover{
	background-color: rgb(213, 190, 147);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/minimizebutton.png</normaloff>../icons/minimizebutton.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pinNote">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: none;
	border-top-left-radius: 8px;
}
QPushButton:hover{
	background-color: rgb(213, 190, 147);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/pin-icon.png</normaloff>../icons/pin-icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pickColor">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: none;
}
QPushButton:hover{
	background-color: rgb(213, 190, 147);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/paint-bucket.png</normaloff>../icons/paint-bucket.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="noteMainFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>19</y>
     <width>400</width>
     <height>280</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="clearNote">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>260</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: none;
	border-bottom-right-radius: 8px;
}
QPushButton:hover{
	background-color: rgb(213, 190, 147);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/clear-icon.png</normaloff>../icons/clear-icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>400</width>
      <height>270</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>270</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>270</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom-right-radius: 8px;
border-bottom-left-radius: 8px;
border: none;</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="lineWrapMode">
     <enum>QPlainTextEdit::NoWrap</enum>
    </property>
    <property name="maximumBlockCount">
     <number>0</number>
    </property>
   </widget>
   <zorder>textBox</zorder>
   <zorder>clearNote</zorder>
  </widget>
  <zorder>noteMainFrame</zorder>
  <zorder>noteTitleFrame</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
